language-design.md

```diff
@@ -17,7 +17,82 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+1. From the article by Pavlus:
+
+>"According to findings by researchers from Southern Illinois University, this
+>reaction isn’t just because you’re a n00b: they found that Perl, a major
+>programming language used by untold zillions of developers, is no more
+>intuitive to novices than a language with a randomly generated syntax."
+>[Pavlus, 2012].
+
+We selected this quote as an example of something we disagree with.  Most
+notably, the quote contradicts itself.  The quote first says that if you
+have trouble intuiting code then it's not because you're a n00b.  However,
+they go on to reference a studying that tested people who have never coded
+before.  This quote does not support their point that experienced coders
+struggle with syntax.
+
+We also disagree with the sentiment that programmers should employ syntax more
+intuitive to novices.  Sure, using natural language could look more
+understandable for someone who has never coded before.  However, for more
+experienced coders, using natural language could feel ambiguous and cumbersome.
+Frequent users of code would appreciate a concise set of keywords that they
+could build off of.  Also, we think it is more important to cater to experienced
+coders than the "n00bs" in the study because they were only novices for an hour.
+Typically, people spend more time as experienced coders who understand the
+syntax than as beginners trying to read off code like a natural language.  There
```

Comment:
Renata makes a great point here! This description is so similar to my own thought process when I read code I just had to jump in and give my experience as well. My brain definitely automatically pattern matches idiomatic for loops into a natural language "do 10 times", while a non-idiomatic one takes a little time to ensure I understand what it is saying.  I suppose one fear here is that my eyes might miss such a subtle difference from an idiomatic for loop, potentially leading me to misread code if I don't see the equal sign.